packs/rabbitmq/actions/publish_message.py

```diff
@@ -0,0 +1,13 @@
+from kombu import BrokerConnection, Exchange, Queue
```

Comment:
Yeah, I chose Kombu since it's required by StackStorm. I'll take a look at switching over to Pika.